### ISSUE TITLE
Can't create VPN Service [Android 12, No root]

### ISSUE THREAD

**Issue by TheDarkKRONOS** [MatsuriDayo/NekoBoxForAndroid#296]
Hello. NekoBox can't create VPN service on OnePlus7 Android 12, no root.

Add VLESS-Reality configuration by QR-code, then tap to bottom button for start. In settings Service Mode -> VPN. In Proxy Mode I can connect by VLESS-Reality and start.

[Error] [VpnRequestActivity$StartService] Failed to start VpnService: null

**Comment by Masteroflose77** [comment]
this is a solotion?

**Comment by TheDarkKRONOS** [comment]
> this is a solotion?

Yes. After deactivate "Always on VPN" in wireguard or other vpn NekoBox can create vpn service on Android

**Comment by Masteroflose77** [comment]
> > this is a solotion?
> 
> Yes. After deactivate "Always on VPN" in wireguard or other vpn NekoBox can create vpn service on Android

Thanks
